draft-ietf-privacypass-architecture.md

```diff
@@ -330,23 +340,23 @@ Each issuance protocol may be different, e.g., in the number and types of
 participants, underlying cryptographic constructions used when issuing tokens,
 and even privacy properties.
 
-Clients initiate the issuance protocol using the challenge, a randomly
+Clients initiate the issuance protocol using the token challenge, a randomly
 generated nonce, and public key for the Issuer, all of which are the Client's
 private input to the protocol and ultimately bound to an output Token;
-see Section 2.2. of {{AUTHSCHEME}} for details. Future specifications
+see {{Section 2.2 of AUTHSCHEME}} for details. Future specifications
 may change or extend the Client's input to the issuance protocol to produce
 Tokens with a different structure.
 
 The issuance protocol itself can be any interactive protocol between Client,
-Issuer, or other parties that produces a valid authenticator over the Client's
+Issuer, or other parties that produces a valid token bound to the Client's
 private input, subject to the following security requirements.
 
 1. Unconditional input secrecy. The issuance protocol MUST NOT reveal anything
 about the Client's private input, including the challenge and nonce, to the
 Attester or Issuer, regardless of the hardness assumptions of the underlying
 cryptographic protocol(s). The issuance protocol can reveal the Issuer public
 key for the purposes of determining which private key to use in producing the
-token.
+token. This property is sometimes also referred to as blindness.
```

Comment:
Do we need to explicitly refer to this property as "blindness" or just leave it as Unconditional input secrecy?

Reply:
Let's leave this as-is.